path-traversal/GnomeGroup_NK-Node/nk.js

```
@@ -0,0 +1,313 @@
+				module.exports.app[type]( path, ( req, res, next ) => callback( res, module.exports.getIP( req ), ( ( !req.body || ( module.exports.stringify( req.body ) == '{}' ) )? ( ( !req.params || ( module.exports.stringify( req.params ) == '{}' ) )? req.query: req.params ): req.body ), req.universalCookies, req.files, req.hostname ) )
+			}
+		}
+	},
+	http: ( secure, webHost, webMethod, webPort, webHeader, webPath, webData, callback ) => {
+		try {
+			let webObj = { host: webHost, port: webPort, method: webMethod, path: webPath }
+			if( webHeader )	{
+				webObj.headers = webHeader
+			}
+			if( webData.formData )	{
+				webData.formData = webData.formData
```

Biome (2.1.2)

[error] formData is assigned to itself.

This is where is assigned.

Self assignments have no effect and can be removed.

(lint/correctness/noSelfAssign)